# Get a servo encoder filter's invert setting. 
int32 servo_id
---
int32 invert